client/cli/go/cmds/cluster.go

```diff
@@ -27,24 +27,24 @@ import (
 )
 
 func init() {
-	RootCmd.AddCommand(clusterCommand)
-	clusterCommand.AddCommand(clusterCreateCommand)
-	clusterCommand.AddCommand(clusterDeleteCommand)
-	clusterCommand.AddCommand(clusterListCommand)
-	clusterCommand.AddCommand(clusterInfoCommand)
-	clusterCreateCommand.SilenceUsage = true
-	clusterDeleteCommand.SilenceUsage = true
-	clusterInfoCommand.SilenceUsage = true
-	clusterListCommand.SilenceUsage = true
+	RootCmd.AddCommand(ClusterCommand)
+	ClusterCommand.AddCommand(ClusterCreateCommand)
```

Comment:
I faced this issue when I tried it in the same directory. I rechecked it too. still moving it to same directory.